R/semanticPage.R

```diff
@@ -26,16 +28,16 @@ getDeps <- function() {
 #' This creates a Semantic page for use in a Shiny app.
 #'
 #' @param title A title to display in the browser's title bar.
-#' @param ... Other arguments to be added as attributes of the main div tag wrapper (e.g. style, class etc.)
+#' @param ... Other arguments to be added as attributes of the main div tag
+#' wrapper (e.g. style, class etc.)
 #'
 #' @export
-semanticPage <- function(..., title = "") {
+semantic_page <- function(..., title = "") {
```

Comment:
this is a change we should give a thought, I think:
1. this is a breaking change, right? So this will break things for current users of the package
2. camelCase was chosen for a reason --- it was to be consistent with Shiny. Why should we change this now?

Reply:
Roled back to camelCase and added `# nolint`